wildfire/lib/screens/all_updates/all_updates.dart

```diff
@@ -0,0 +1,352 @@
+                          ),
+                          SizedBox(
+                            width: 343.w,
+                            height: 21.h,
+                          ),
+                          SizedBox(height: 8.h),
+                          Container(
+                            width: 343.w,
+                            height: 165.h,
+                            decoration: BoxDecoration(
+                              borderRadius: BorderRadius.circular(24.r),
+                              color: Color.fromRGBO(236, 236, 236, 1),
+                            ),
+                            padding: EdgeInsets.symmetric(
+                              horizontal: 16.w,
+                            ),
+                            child: Row(
+                              mainAxisSize: MainAxisSize.min,
+                              mainAxisAlignment: MainAxisAlignment.end,
+                              crossAxisAlignment: CrossAxisAlignment.center,
+                              children: [
+                                Container(
+                                  child: Icon(
+                                    Icons.location_on_outlined,
+                                    size: 12.r,
+                                    color: Color.fromRGBO(121, 116, 116, 1),
+                                  ),
+                                ),
+                                Text(
+                                  "Cleveland",
+                                  style: GoogleFonts.openSans(
+                                    textStyle: TextStyle(
+                                      color: Color(0xff797474),
+                                      fontSize: 10.sp,
+                                    ),
+                                  ),
+                                ),
+                                SizedBox(width: 5.w),
+                                Container(
+                                  width: 3.w,
+                                  height: 3.h,
+                                  decoration: BoxDecoration(
+                                    shape: BoxShape.circle,
+                                    color: Color(0xff797474),
+                                  ),
+                                ),
+                                SizedBox(width: 5.w),
+                                Text(
+                                  "1h ago",
+                                  style: GoogleFonts.openSans(
+                                    textStyle: TextStyle(
+                                      color: Color(0xff797474),
+                                      fontSize: 10.sp,
+                                    ),
+                                  ),
+                                ),
+                                SizedBox(
+                                  width: 157.w,
+                                  height: 140.h,
+                                  child: Text(
```

Comment:
This text is supposed to be coming from the index not hardcoded same as the other widgets